python/packages/autogen-core/src/autogen_core/store/abstract_store_base.py

```diff
@@ -0,0 +1,34 @@
+from typing import Any, Optional, Protocol
+
+
+class AbstractStore(Protocol):
```

Comment:
The naming convention in this repo has been to not use "Abstract", so just `CacheStore`.

Reply:
We can have the protocol in `autogen_core.CacheStore` like all other base components.